x/projects/keeper/project.go

```diff
@@ -4,11 +4,55 @@ import (
 	"fmt"
 	"strconv"
 
+	"golang.org/x/exp/slices"
+
+	"github.com/cosmos/cosmos-sdk/store/prefix"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/lavanet/lava/utils"
 	"github.com/lavanet/lava/x/projects/types"
 )
 
+func (k Keeper) GetSubscriptionProjects(ctx sdk.Context, subscriptionAddr string) []string {
```

Comment:
we dont need all this functionality, just iterate on all the fixations with the subscription prefix, this is redundent